lambda/utilities/vector_store.py

```diff
@@ -34,12 +34,18 @@
 secretsmanager_client = boto3.client("secretsmanager", region_name=os.environ["AWS_REGION"], config=retry_config)
 
 
-def get_vector_store_client(store: str, index: str, embeddings: Embeddings) -> VectorStore:
+def get_vector_store_client(
```

Comment:
Can you get the repo_type from the repo_id? We use this get_vector_store_client a bunch, and the API endpoints all pass in a repo_type, which wouldn't be needed if we are fully using repo_id.